internal/saucecloud/imagerunner.go

```diff
@@ -200,6 +215,19 @@ func (r *ImgRunner) buildService(serviceIn imagerunner.SuiteService, suiteName s
 	return serviceOut, nil
 }
 
+func ignoreError(err error) bool {
+	if err == nil {
+		return true
+	}
+	if !errors.Is(err, context.Canceled) {
+		return true
+	}
+	if strings.Contains(err.Error(), "websocket: close") {
+		return true
+	}
+	return false
+}
```

Comment:
Considering that this is a function to be used somewhere and is defined at the package level, it kinda lacks context as to when it should be used (i.e. its name is too generic, suggesting that it could be used for any error check in this package). Has no docs either. If https://github.com/saucelabs/saucectl/blob/e166ab0f1be444027f9e98f6e85e9526b58696b5/internal/saucecloud/imagerunner.go#L309-L309 is the only place, it may have been better as a inlined function.